### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JacquelineMorrissette**
Updated the README.md to include the steps to run the tests in the KoBoCat container to limit time spent troubleshooting for anyone who may not be too familiar with the code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,3 +76,26 @@ To compile MO files and update live translations
...
+Enter the container
+
+``` sh
+$ docker exec -it {{kobocat container}} /bin/bash
+```
+
+Run pip install the development dependancies
+
+``` sh
+$ pip insall -r requirements/dev.pip
```
Comment (noliveleger) [inline README.md:90]: typo: `install`

File: README.md [hunk 2]
```diff
@@ -76,3 +76,26 @@ To compile MO files and update live translations
 $ django-admin.py compilemessages ;
 $ for app in {main,viewer} ; do cd kobocat/apps/${app} && django-admin.py compilemessages && cd - ; done
 ```
+## Testing in Kobocat
+
+Enter the container
+
+``` sh
+$ docker exec -it {{kobocat container}} /bin/bash
```
Comment (noliveleger) [inline README.md:84]: You could add command with kobo-install too. (i.e. `For kobo-install users `./run.py -cf exec kobocat bash`)

**Review by noliveleger — CHANGES_REQUESTED** [review]

**Review by JacquelineMorrissette — COMMENTED** [review]
@noliveleger I made both the changes you suggested. How does it look now?

File: README.md [hunk 3]
```diff
@@ -76,3 +76,33 @@ To compile MO files and update live translations
 $ django-admin.py compilemessages ;
 $ for app in {main,viewer} ; do cd kobocat/apps/${app} && django-admin.py compilemessages && cd - ; done
 ```
+## Testing in Kobocat
+
+For Kobo-Install Users, enter the folder for Kobo-Install and run this command
```
Comment (noliveleger) [inline README.md:81]: I think the correct way is `KoBoCAT` (see https://github.com/kobotoolbox/kobocat/blob/master/README.md) 
I used to write KoBo-Install, but we decided to change to `kobo-install` (https://github.com/kobotoolbox/kobo-install/commit/b4da7d7415af60ab2c299f1a4aeb011b76b5dcaa) to match with `kobo-docker`

It's not an error at all but I think it's better to be consistent in every repo. What do you think?

**Review by noliveleger — APPROVED** [review]

**Pull request merged**